main.go

```diff
@@ -89,9 +108,11 @@ func main() {
 		os.Exit(1)
 	}
 
+	go runPrometheusServer(metricsAddr)
+
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
 		Scheme:                 scheme,
-		MetricsBindAddress:     metricsAddr,
+		MetricsBindAddress:     "0",
```

Comment:
That makes us loose existing metrics exported by controller-runtime :-/
Can we merge those metrics somehow so we add to them instead of replacing them?

Reply:
done.

Reply:
Thanks, that looks better!